src/docfx/cli/CommandLineOptions.cs

```diff
@@ -31,6 +31,8 @@ internal class CommandLineOptions
 
         public bool LanguageServer { get; init; }
 
+        public int? Port { get; init; }
```

Comment:
We'll probably need 3 command line args for this, [v2 has 2](https://github.com/dotnet/docfx/blob/dev/src/docfx/Models/ServeCommandOptions.cs):

- `--port`: Port to use (defaults to 8080)
- `--address`: Address to use (defaults to 0.0.0.0)
- `--https`: Enables https (defaults to false)